run.py

```
@@ -25,436 +26,449 @@
+                if epoch > cos_epoch:
+                    lr_schedule.step()
+
+            local_loss = loss.item()
+            total_loss += local_loss
+            tbar.set_description(f"Epoch {epoch + 1} Loss: {total_loss/(idx+1)}")
+
+            if accelerator.is_local_main_process:
+                if step % 1000 == 0:
+                    wandb.log(
+                        {
+                            "train_loss": train_loss / (idx + 1),
```

Ruff (0.8.2)

Undefined name `train_loss`

(F821)